examples/alerting_example_test.go

```diff
@@ -59,8 +59,8 @@ func runCreateManualMitigation() error {
 	// This data is invalid and saved filter will not be created.
 	mm := models.ManualMitigation{
 		IPCidr:                "192.168.0.0/24",
-		PlatformID:            1234,
-		MethodID:              12345,
+		PlatformID:            "1234",
```

Comment:
Maybe we should improve consistency of our examples and make it models.ID("1234") like below in examples/demos/devices_users_demo/main.go
?

Reply:
I'd rather simplify initializations to omit models.ID type.

Pros: code is simpler
Cons: if we decide to convert models.ID to be custom type (not type alias), code stops working. IMO chances that we need that are minimal.

Reply:
This initialization only works because ID is alias to string, eventually it should become a separate domain type of it's own

Reply:
Why eventually it should become a separate domain type? What value that would bring?

Reply:
Compiler protects the user from inputting any string by mistake

Reply:
That's a fair point. It helps avoiding this type of bugs. But adds a little bit of complexity.